crates/bin/pcli/tests/network_integration.rs

```diff
@@ -36,14 +36,16 @@ const TEST_ASSET: &str = "1020test_usd";
 const TIMEOUT_COMMAND_SECONDS: u64 = 20;
 
 // The time to wait before attempting to perform an undelegation claim.
-// By default the epoch duration is 100 blocks, the block time is ~500 ms,
-// and the number of unbonding epochs is 2.
+// The "unbonding_delay" value is specified in blocks, and in the smoke tests,
+// block time is set to ~500ms, so we'll take the total number of blocks
+// that must elapse and sleep half that many seconds.
 static UNBONDING_DURATION: Lazy<Duration> = Lazy::new(|| {
-    let blocks: f64 = std::env::var("EPOCH_DURATION")
+    let blocks: f64 = std::env::var("UNBONDING_DELAY")
```

Comment:
checking my understanding, this is fixing a pre-existing bug unrelated to the changes in this branch?

Reply:
It's more of an optimization, but that's right: we could have (and should have) merged this isolated change without rewriting the smokes, to shave a few minutes off the run time. Improving the ergonomics of running the tests locally made it obvious this runtime was longer than it should have been, which motivated me to explore the pcli sleeps. I'm sure there are opportunities for further refinement, but not pursuing them for now, to focus on more pressing work.

Reply:
thank you for the color :art: :paintbrush: